release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1309,6 +1313,63 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
++
+* `--retry-times`: Specifies the number of retry attempts. Default is 2.
+* `--retry-delay`: Sets the delay between retries. Default is 1 second.
+* `--image-timeout`: Defines the timeout period for mirroring an image. Default is 10 minutes.
+* `--max-parallel-downloads`: Controls the maximum number of layers to pull simultaneously during a single copy operation. Default is 6.
++
+(link:https://issues.redhat.com/browse/OCPBUGS-34021[*OCPBUGS-34021*])
+
+* Previously, the technology preview oc-mirror plug-in v2 feature did not generate `ImageDigestMirrorSet` (IDM) or `ImageTagMirrorSet` (ITM) files when mirroring failed due to network issues or invalid operator catalogs. With this update, oc-mirror continues mirroring other images when operator or additional images fail, and only stops when release images fail. Cluster resources are generated based on successfully mirrored images, and all errors are collected in a log file for review.
+(link:https://issues.redhat.com/browse/OCPBUGS-34020[*OCPBUGS-34020*])
+
+* Previously, in the technology preview oc-mirror plug-in v2 feature, there was no validation for the `-v2` vs `--v2` flags. As a result, users who mistakenly used `-v2`, that sets the log level to 2 instead of `--v2`, that switches to oc-mirror v2, encountered unclear error messages. With this update, flag validation is provided. If `-v2` is used while the `ImageSetConfig` is using the `v2alpha1` API and `--v2` is not specified, an error message is displayed. The following message is now enabled that provides a clear guidance to the user:
++
+[source,terminal]
+----
+[ERROR]: Detected a v2 `ImageSetConfiguration`, please use `--v2` instead of `-v2`.
+----
++
+(link:https://issues.redhat.com/browse/OCPBUGS-33121[*OCPBUGS-33121*])
+
+* Previously, the technology preview oc-mirror plug-in v2 feature would stop the mirroring process when it encountered operators using bundle versions that were not compliant with semantic versioning, which also prevented the creation of cluster resources like IDMS, ITMS, and `CatalogSource`. With this fix, oc-mirror v2 now skips these problematic images instead of halting the process. If an image uses incorrect semantic versioning, a warning message is displayed in the console with the relevant image details. (link:https://issues.redhat.com/browse/OCPBUGS-33081[*OCPBUGS-33081*])
+
+* Previously, the `oc set env` command wrongly changed the apiVersion of Route and `DeploymentConfig` objects, removing the group part. For example, `apps.openshift.io/v1` became `v1`. This caused the command to exit with "unable to recognize no matches for kind" errors. With this release, the error is fixed so that the `os set env` command keeps the correct version in the Route and `DeploymentConfig` objects. (link:https://issues.redhat.com/browse/OCPBUGS-32108[*OCPBUGS-32108*])
```

Comment:
```suggestion
* Previously, the `oc set env` command incorrectly changed the API version of `Route` and `DeploymentConfig` objects, for example, `apps.openshift.io/v1` became `v1`. This caused the command to exit with _unable to recognize no matches for kind_ errors. With this release, the error is fixed so that the `os set env` command keeps the correct API version in `Route` and `DeploymentConfig` objects. (link:https://issues.redhat.com/browse/OCPBUGS-32108[*OCPBUGS-32108*])
```